source/administration/config-in-database.rst

```diff
@@ -0,0 +1,26 @@
+Configuration in the Mattermost Database
+=========================================
+A new configuration database was added in `5.10 <https://docs.mattermost.com/administration/changelog.html#configuration-in-database>`_ to allow using the database for the single source of truth of the active configuration of your Mattermost installation. This provides a convenient way to manage configuration changes from the system console and provides a failsafe in high availability deployments when a server in the node goes down.  Additionally, in high availability deployments, configuration changes are applied to all nodes instantaneously as possible removing the need to update individual configuration files on each server and the read-only state of the system console is removed.   
```

Comment:
Do you mean `A new configuration option`

Reply:
- `system console` -> `System Console`
- `failsafe` may not be obvious for non-English native spakers

Reply:
Wondering if there's a simpler way to write the last sentence. Or maybe having the benefits in bullets helps with readability

Reply:
I have addressed this feedback.

Reply:
There's one more `system console` -> `System Console` 🙏

Reply:
Thanks - missed that one.